Configuration checks for too low Pipewire version

**mpv version**: 050f5e7915045a28ada36c55bf817b8f909020ae
**build system** tested waf, but meson appears to be affected as well

### Reproduction steps

Have libpipewire-0.3 development files installed, for a version lower  than `0.3.44` and run `./waf configure`. *(In my case it's `0.3.19`.)* It will autodetect Pipewire support, but during building fail with
```
../audio/out/ao_pipewire.c: In function ‘init’:
../audio/out/ao_pipewire.c:464:9: error: ‘PW_KEY_TARGET_OBJECT’ undeclared (first use in this function)
  464 |         PW_KEY_TARGET_OBJECT, ao->device,
      |         ^~~~~~~~~~~~~~~~~~~~
../audio/out/ao_pipewire.c:464:9: note: each undeclared identifier is reported only once for each function it appears in
```

This macro was added in [pipewire@9935f921c](https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/9935f921c512f1b3eec9f1608e86d13560ffa27c) first contained in release `0.3.44`.

### Expected behavior

Either raise the version checked during configuration or `#ifdef` usage of newer symbols than what's checked in configuration.
If raising the checked version to `0.3.44` (effectively disabling pipewire) the build succeeds again for me.

### Relevant Config Log file excerpt
```
Checking for PipeWire audio output
['/usr/bin/pkg-config', 'libpipewire-0.3 >= 0.3.0', '--libs', '--cflags', 'libpipewire-0.3']
out: -D_REENTRANT -I/usr/include/pipewire-0.3 -I/usr/include/spa-0.2 -lpipewire-0.3

yes
```

--- a/audio/out/ao_pipewire.c
+++ b/audio/out/ao_pipewire.c
@@ -42,6 +42,12 @@
 #define PW_KEY_NODE_RATE "node.rate"
 #endif
 
+// Added in Pipewire 0.3.44
+// remove the fallback when we require a newer version
+#ifndef PW_KEY_TARGET_OBJECT
+#define PW_KEY_TARGET_OBJECT "target.object"
+#endif
+
 #if !PW_CHECK_VERSION(0, 3, 50)
 static inline int pw_stream_get_time_n(struct pw_stream *stream, struct pw_time *time, size_t size) {
 	return pw_stream_get_time(stream, time);

--- a/ci/build-tumbleweed.sh
+++ b/ci/build-tumbleweed.sh
@@ -9,6 +9,7 @@ if [ "$1" = "meson" ]; then
       -Dlibarchive=enabled    \
       -Dlibmpv=true           \
       -Dmanpage-build=enabled \
+      -Dpipewire=enabled      \
       -Dshaderc=enabled       \
       -Dtests=true            \
       -Dvulkan=enabled
@@ -24,6 +25,7 @@ if [ "$1" = "waf" ]; then
       --enable-libarchive    \
       --enable-libmpv-shared \
       --enable-manpage-build \
+      --enable-pipewire      \
       --enable-shaderc       \
       --enable-tests         \
       --enable-vulkan

--- a/meson.build
+++ b/meson.build
@@ -831,7 +831,7 @@ if features['oss-audio']
     sources += files('audio/out/ao_oss.c')
 endif
 
-pipewire = dependency('libpipewire-0.3', version: '>= 0.3', required: get_option('pipewire'))
+pipewire = dependency('libpipewire-0.3', version: '>= 0.3.19', required: get_option('pipewire'))
 features += {'pipewire': pipewire.found()}
 if features['pipewire']
     dependencies += pipewire

--- a/wscript
+++ b/wscript
@@ -446,7 +446,7 @@ audio_output_features = [
     }, {
         'name': '--pipewire',
         'desc': 'PipeWire audio output',
-        'func': check_pkg_config('libpipewire-0.3', '>= 0.3.0')
+        'func': check_pkg_config('libpipewire-0.3', '>= 0.3.19')
     }, {
         'name': '--sndio',
         'desc': 'sndio audio input/output',

